Index: 확인.html
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><!DOCTYPE html>\r\n<html lang=\"en\">\r\n<head>\r\n    <meta charset=\"UTF-8\">\r\n    <meta http-equiv=\"X-UA-Compatible\" content=\"IE=edge\">\r\n    <meta name=\"viewport\" content=\"width=device-width, initial-scale=1.0\">\r\n    <title>오늘 해야 할 것</title>\r\n</head>\r\n<body>\r\n<script>\r\n    getDaystudent(39);\r\n\r\n    function getDaystudent( day ){\r\n        document.write(\"오늘은 \"+day+\"일차<br>\");\r\n        document.write(\"오늘은 \"+(day-1)+\"일차 공부해야합니다<br>\");\r\n        document.write(\"오늘은 \"+(day-3)+\"일차 공부해야합니다<br>\");\r\n        document.write(\"오늘은 \"+(day-6)+\"일차 공부해야합니다<br>\");\r\n        document.write(\"오늘은 \"+(day-13)+\"일차 공부해야합니다<br>\");\r\n        document.write(\"오늘은 \"+(day-29)+\"일차 공부해야합니다<br>\");\r\n    }\r\n    \r\n</script>\r\n    \r\n</body>\r\n</html>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/확인.html b/확인.html
--- a/확인.html	(revision 03c69bd7bd6ee883d8839ef5f5f5dabd7958fde7)
+++ b/확인.html	(date 1657801755186)
@@ -8,7 +8,7 @@
 </head>
 <body>
 <script>
-    getDaystudent(39);
+    getDaystudent(40);
 
     function getDaystudent( day ){
         document.write("오늘은 "+day+"일차<br>");
